src/views/v2/rsvp/ari/form/fields.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This template renders the RSVP AR form fields.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/form/fields.php
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @var Tribe__Tickets__Ticket_Object $rsvp The rsvp ticket object.
+ * @var int $post_id The post ID the RSVP is linked to.
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+$has_meta = get_post_meta( $rsvp->ID, '_tribe_tickets_meta_enabled', true );
```

Comment:
Should this be a filter that ET+ hooks into so this is logic/meta key is not stuck inside a view?